pop11_grammar_tabatkins.txt

```diff
@@ -36,7 +36,8 @@ Diagram(
 			NonTerminal('sequence'),
 			Terminal(')')
 		),
-		NonTerminal('block')
+		NonTerminal('block'),
+		NonTerminal('ident expression')

```

Comment:
We are missing the definition of what an ident expression is i.e. IdentExpression ::= 'ident' Word

Reply:
Right, I don't know how I missed that 😓. Thanks for the review!